### ISSUE TITLE
[BUG] Please do not use deprecated flags.

### ISSUE THREAD

**Issue by mitoma** [DataDog/datadog-agent#15529]
Official Dockerimage use deprecated flag `-config`.
Please use `--cfgpath` instead.

https://github.com/DataDog/datadog-agent/blob/975738dbf7b054cb8ee438d14c6ec5081c415dab/Dockerfiles/agent/entrypoint.d/process-agent#L4

**Agent Environment**

I tried with `public.ecr.aws/datadog/agent:7.42.1-jmx`

**Describe what happened:**

WARNING log appears.
```
WARNING: `--config` argument is deprecated and will be removed in a future version. Please use `--cfgpath` instead.
```

**Describe what you expected:**]

No WARNING log.

**Steps to reproduce the issue:**

Use process-agent on official images.

**Additional environment details (Operating System, Cloud provider, etc):**

**Comment by robertjli** [comment]
Hello @mitoma,

Thank you for reporting this issue! We have merged a fix for it. Since the release process for 7.43 has already started, this fix will be released in 7.44.

I'll leave this issue open until 7.44 is released, so that we can verify it is fixed.

**Comment by robertjli** [comment]
This has been fixed in 7.44.0, but since then a different config was deprecated.

```
2023-05-01 15:22:49 UTC | PROCESS | INFO | (pkg/util/log/log.go:585 in func1) | process_config.enabled is deprecated, use process_config.container_collection.enabled and process_config.process_collection.enabled instead, see https://docs.datadoghq.com/infrastructure/process#installation for more information
```

However, I will close this issue and track the new one separately.

**Comment by apoorva11029** [comment]
Hi @robertjli ! Curious if you created a new issue for the process_config warning and if it's still a known issue? We are seeing it in the latest installation and couldn't figure out if we are misconfiguring the helm chart. Thank you!
